### PR TITLE
Add UUID generator

### PR DESCRIPTION
**Pull request by aarreedd**
Added link to [uuidtools.com/docs](https://www.uuidtools.com/docs). Open source project provides API to generate UUIDs. Supports multiple UUID versions, bulk UUID generation, and UUID decoding. 

---

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -820,6 +820,7 @@ API | Description | Auth | HTTPS | CORS |
 | [RandomUser](https://randomuser.me) | Generates random user data | No | Yes | Unknown |
 | [RoboHash](https://robohash.org/) | Generate random robot/alien avatars | No | Yes | Unknown |
 | [This Person Does not Exist](https://thispersondoesnotexist.com) | Generates real-life faces of people who do not exist | No | Yes | Unknown |
+| [UUID Generator](https://www.uuidtools.com/docs) | Generate UUIDs | No | Yes | No |
```
Comment (matheusfelipeog) [inline README.md:823]: The `uuidtools.com/docs` uses cloudflare service, this broken our tests.

The solution I see to solve this without having to change anything in your project is to add the repository link on github.

Can you do that, please?

```suggestion
| [UUID Generator](https://github.com/aarreedd/uuidtools.com) | Generate UUIDs | No | Yes | No |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for contribution @aarreedd 😄

**Comment by aarreedd** [comment]
I've changed the cloudflare settings. I think the test will pass now.

**Comment by matheusfelipeog** [comment]
Oh, right. Thanks for the change ;)

**Review by matheusfelipeog — APPROVED** [review]

**Pull request merged**